### PR TITLE
04_DNN_VLE: fix train_test_split 

### PR DESCRIPTION
**Pull request by carolineganzer**
Split the test set, then from the rest split the validation set. This removes the current overlap between train, validation, and test sets.

Before:
`    comp_train_all, comp_test = train_test_split(unique_compounds_cl, test_size=0.2, random_state=0)`
`    comp_train, comp_valid = train_test_split(unique_compounds_cl, test_size=0.2, random_state=1)`

After:
`    comp_train_all, comp_test = train_test_split(unique_compounds_cl, test_size=0.2, random_state=0)`
`    comp_train, comp_valid = train_test_split(comp_train_all, test_size=0.2, random_state=1)`

This reduces the apparent performance of the ANNs, but not by much.

### CODE REVIEW

**Comment by edgarsmdn** [comment]
Thanks for catching this! Fixed!
